tests/test_rocket.py

```diff
@@ -489,6 +490,28 @@ def test_add_cp_eccentricity_assert_properties_set(calisto):
     assert calisto.cp_eccentricity_y == 5
 
 
+def test_add_motor(calisto_motorless, cesaroni_m1670):
+    """Tests the add_motor method of the Rocket class.
+    Both with respect to return instances and expected behaviour.
+    Parameters
+    ----------
+    calisto_motorless : Rocket instance
+        A predefined instance of a Rocket without a motor, used as a base for testing.
+    cesaroni_m1670 : rocketpy.SolidMotor
+        Cesaroni M1670 motor
+    """
+    assert isinstance(calisto_motorless.motor, EmptyMotor)
+    center_of_mass_motorless = calisto_motorless.center_of_mass
+    calisto_motorless.add_motor(cesaroni_m1670, 0)
+
+    assert calisto_motorless.motor is not None
```

Comment:
More import to check if it is an instance of the motor class.

Please remeber to import the Motor class at the top of the file.

```suggestion
    assert isinstance(calisto_motorless.motor, Motor)
```

Reply:
Also agree, done it.